### PR TITLE
Documentation clean up 

### PR DESCRIPTION
**Pull request by jordynmackool**
Fixes issue #1787 

### Risk
This PR makes **[no]** API changes.

### Summary
Resolves typos as listed in issue #1787 

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Review by theresalech — APPROVED** [review]
This looks good to me! @joeljfischer, can you please merge at your convenience?

File: docs/Classes/SDLAddCommand.html [hunk 1]
```diff
@@ -403,7 +403,7 @@ <h3 id="section-cmdIcon">
   
   <p>Image struct containing a static or dynamic icon.</p>
 
-<p>@discussion If provided, defines the image to be be shown along with a command. If omitted on supported displays, no (or the default if applicable) icon will be displayed</p>
+<p>@discussion If provided, defines the image to be shown along with a command. If omitted on supported displays, no (or the default if applicable) icon will be displayed</p>
```
Comment (theresalech) [inline docs/Classes/SDLAddCommand.html:406]:
```suggestion
<p>@discussion If provided, defines the image to be shown along with a command. If omitted on supported displays, no (or the default if applicable) icon will be displayed.</p>
```

File: docs/Classes/SDLDisplayCapability.html [hunk 2]
```diff
@@ -69,7 +69,7 @@ <h3 id="section--initWithDisplayName:windowCapabilities:windowTypeSupported:">
       -initWithDisplayName:windowCapabilities:windowTypeSupported:
   </h3>
   
-  <p>Init with all the properities</p>
+  <p>Init with all the prosperities</p>
```
Comment (theresalech) [inline docs/Classes/SDLDisplayCapability.html:72]:
```suggestion
  <p>Init with all the properties</p>
```

File: docs/Classes/SDLPerformInteraction.html [hunk 3]
```diff
@@ -146,7 +146,7 @@ <h3 id="section--initWithInteractionChoiceSetId:">
       -initWithInteractionChoiceSetId:
   </h3>
   
-  <p>Convenience init for setting the a single visual or voice-recognition menu choice.</p>
+  <p>Convenience init for setting the single visual or voice-recognition menu choice.</p>
```
Comment (theresalech) [inline docs/Classes/SDLPerformInteraction.html:149]:
```suggestion
  <p>Convenience init for setting a single visual or voice-recognition menu choice.</p>
```

File: docs/Classes/SDLScreenManager.html [hunk 4]
```diff
@@ -351,7 +351,7 @@ <h3 id="section-dynamicMenuUpdatesMode">
 
 <p>The current status for dynamic menu updates. A dynamic menu update allows for smarter building of menu changes. If this status is set to <code>SDLDynamicMenuUpdatesModeForceOn</code>, menu updates will only create add commands for new items and delete commands for items no longer appearing in the menu. This helps reduce possible RPCs failures as there will be significantly less commands sent to the HMI.</p>
 
-<p>If set to <code>SDLDynamicMenuUpdatesModeForceOff</code>, menu updates will work the legacy way. This means when a new menu is set the entire old menu is deleted and add commands are created for every item regarldess if the item appears in both the old and new menu. This method is RPCs heavy and may cause some failures when creating and updating large menus.</p>
+<p>If set to <code>SDLDynamicMenuUpdatesModeForceOff</code>, menu updates will work the legacy way. This means when a new menu is set the entire old menu is deleted and add commands are created for every item regardless if the item appears in both the old and new menu. This method is RPCs heavy and may cause some failures when creating and updating large menus.</p>
```
Comment (theresalech) [inline docs/Classes/SDLScreenManager.html:354]:
```suggestion
<p>If set to <code>SDLDynamicMenuUpdatesModeForceOff</code>, menu updates will work the legacy way. This means when a new menu is set, the entire old menu is deleted and add commands are created for every item regardless of if the item appears in both the old and new menu. This method is RPCs heavy and may cause some failures when creating and updating large menus.</p>
```

File: docs/Classes/SDLSeatControlCapabilities.html [hunk 5]
```diff
@@ -136,7 +136,7 @@ <h4>Parameters</h4>
       <dt>coolingLevelAvail</dt>
       <dd><p>Whether or not cooling level is available</p></dd>
       <dt>horizontalPositionAvail</dt>
-      <dd><p>Whether or not horizontal Position is aavailable</p></dd>
+      <dd><p>Whether or not horizontal Position is available</p></dd>
```
Comment (theresalech) [inline docs/Classes/SDLSeatControlCapabilities.html:139]:
```suggestion
      <dd><p>Whether or not horizontal position is available</p></dd>
```

File: docs/Classes/SDLStreamingMediaConfiguration.html [hunk 6]
```diff
@@ -110,7 +110,7 @@ <h3 id="section-rootViewController">
 
 </div><div class="aside aside-warning">
     <p class="aside-title">Warning</p>
-    <p>Apps using views outside of the <code>UIView</code> heirarchy (such as OpenGL) are currently unsupported. If you app uses partial views in the heirarchy, only those views will be discovered. Your OpenGL views will not be discoverable to a haptic interface head unit and you will have to manually make these views discoverable via the <code><a href="../Classes/SDLSendHapticData.html">SDLSendHapticData</a></code> RPC request.</p>
+    <p>Apps using views outside of the <code>UIView</code> hierarchy (such as OpenGL) are currently unsupported. If you app uses partial views in the hierarchy, only those views will be discovered. Your OpenGL views will not be discoverable to a haptic interface head unit and you will have to manually make these views discoverable via the <code><a href="../Classes/SDLSendHapticData.html">SDLSendHapticData</a></code> RPC request.</p>
```
Comment (theresalech) [inline docs/Classes/SDLStreamingMediaConfiguration.html:113]:
```suggestion
    <p>Apps using views outside of the <code>UIView</code> hierarchy (such as OpenGL) are currently unsupported. If your app uses partial views in the hierarchy, only those views will be discovered. Your OpenGL views will not be discoverable to a haptic interface head unit and you will have to manually make these views discoverable via the <code><a href="../Classes/SDLSendHapticData.html">SDLSendHapticData</a></code> RPC request.</p>
```

File: docs/Classes/SDLStreamingMediaConfiguration.html [hunk 7]
```diff
@@ -263,11 +263,11 @@ <h4>Parameters</h4>
       <dt>securityManagers</dt>
       <dd><p>The security managers to use or nil for none.</p></dd>
       <dt>encryptionFlag</dt>
-      <dd><p>The maximum encrpytion supported. If the connected head unit supports less than set here, it will still connect, but if it supports more than set here, it will not connect.</p></dd>
+      <dd><p>The maximum encryption supported. If the connected head unit supports less than set here, it will still connect, but if it supports more than set here, it will not connect.</p></dd>
       <dt>videoSettings</dt>
       <dd><p>Custom video encoder settings to be used in video streaming.</p></dd>
       <dt>rootViewController</dt>
-      <dd><p>The UIViewController wih the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews)</p></dd>
+      <dd><p>The UIViewController with the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews)</p></dd>
```
Comment (theresalech) [inline docs/Classes/SDLStreamingMediaConfiguration.html:270]:
```suggestion
      <dd><p>The UIViewController with the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews).</p></dd>
```

File: docs/Classes/SDLStreamingMediaConfiguration.html [hunk 8]
```diff
@@ -297,11 +297,11 @@ <h4>Objective-C</h4>
   <h4>Parameters</h4>
   <dl>
       <dt>encryptionFlag</dt>
-      <dd><p>The maximum encrpytion supported. If the connected head unit supports less than set here, it will still connect, but if it supports more than set here, it will not connect.</p></dd>
+      <dd><p>The maximum encryption supported. If the connected head unit supports less than set here, it will still connect, but if it supports more than set here, it will not connect.</p></dd>
       <dt>videoSettings</dt>
       <dd><p>Custom video encoder settings to be used in video streaming.</p></dd>
       <dt>rootViewController</dt>
-      <dd><p>The UIViewController wih the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews)</p></dd>
+      <dd><p>The UIViewController with the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews)</p></dd>
```
Comment (theresalech) [inline docs/Classes/SDLStreamingMediaConfiguration.html:304]:
```suggestion
      <dd><p>The UIViewController with the content that is being streamed on, to use for haptics if needed and possible (only works for UIViews).</p></dd>
```

File: docs/Classes/SDLTextField.html [hunk 9]
```diff
@@ -112,7 +112,7 @@ <h3 id="section--initWithName:characterSet:width:rows:">
       -initWithName:characterSet:width:rows:
   </h3>
   
-  <p>Convenience initalizer for the TextField RPC struct</p>
+  <p>Convenience initializer for the TextField RPC struct</p>
```
Comment (theresalech) [inline docs/Classes/SDLTextField.html:115]:
```suggestion
  <p>Convenience initializer for the TextField RPC struct.</p>
```

File: docs/Enums/SDLSystemCapabilityManagerError.html [hunk 10]
```diff
@@ -36,7 +36,7 @@ <h3 id="section-SDLSystemCapabilityManagerErrorHMINone">
       SDLSystemCapabilityManagerErrorHMINone
   </h3>
   
-  <p>You must be in an HMI Level other than NONE to request or subscribe to capabilites other than DISPLAYS</p>
+  <p>You must be in an HMI Level other than NONE to request or subscribe to capabilities other than DISPLAYS</p>
```
Comment (theresalech) [inline docs/Enums/SDLSystemCapabilityManagerError.html:39]:
```suggestion
  <p>You must be in an HMI level other than NONE to request or subscribe to capabilities other than DISPLAYS.</p>
```

**Review by theresalech — CHANGES_REQUESTED** [review]
@jordynmackool, I've suggested a few changes. Please let me know if you have any questions!

**Comment by jordynmackool** [comment]
@theresalech, thanks for the review. I have made committed your suggested changes.

**Review by theresalech — APPROVED** [review]
Looks good to me! @joeljfischer, please merge when you get the chance!

**Pull request merged**